Source/JavaScriptCore/runtime/ProxyObject.cpp

```diff
@@ -834,7 +834,7 @@ bool ProxyObject::performDefineOwnProperty(JSGlobalObject* globalObject, Propert
 
     if (!trapResultAsBool) {
         if (shouldThrow)
-            throwTypeError(globalObject, scope, makeString("Proxy's 'defineProperty' trap returned falsy value for property '", String(propertyName.uid()), "'"));
+            throwTypeError(globalObject, scope, makeString("Proxy's 'defineProperty' trap returned falsy value for property '"_s, String(propertyName.uid()), '\''));
```

Comment:
Should be able to pass `propertyName.uid()` without `String` to avoid reference count churn.